src/components/userProfile/index.jsx

```diff
@@ -7,10 +7,11 @@ import { CheckOutlined, CloseOutlined } from '@ant-design/icons';
 function UserProfilePage() {
 
     const [userData, setUserData] = useState({});
-    const [fullName, setFullName] = useState("Undefined");
+    const [fullName, setFullName] = useState();
```

Comment:
I suppose you can avoid creating this state by using the userData directly in the DOM in 55 line.